AsyncDisplayKit/ASDisplayNodeExtras.mm

```diff
@@ -251,13 +251,19 @@ static inline BOOL _ASDisplayNodeIsAncestorOfDisplayNode(ASDisplayNode *possible
 
 extern UIWindow * _Nullable ASFindWindowOfLayer(CALayer *layer)
 {
+  UIView *view = ASFindViewOfLayer(layer);
```

Comment:
I tend to like `ASFindClosestViewOfLayer` actually. Definitely not a blocking issue though.